packages/react/src/PageHeader/PageHeader.tsx

```diff
@@ -754,6 +762,7 @@ const Navigation: React.FC<React.PropsWithChildren<NavigationProps>> = ({
       aria-label={as === 'nav' ? ariaLabel : undefined}
       aria-labelledby={as === 'nav' ? ariaLabelledBy : undefined}
       className={clsx(enabled && classes.Navigation, className)}
+      data-component="PH_Navigation"
```

Comment:
```suggestion
      data-component="PageHeader.Navigation"
```

One suggestion that could make it more clear what the component is when coming back to this later

Reply:
`PH_{Subcomponent}` is the naming pattern we use for the other subcomponents. Do you still think we should change it?

Reply:
I think we're kind of trying to move away from `data-component` actually.. what about just giving it a classname instead?

Reply:
I agree, `PageHeader.Navigation` is more clear, but to keep it consistent I think `PH_{Subcomponent}` is fine too! Would be curious about the className route too 👀